.github/workflows/nightly.yml

```
@@ -9,6 +9,8 @@ jobs:
     name: 🌒 Nightly Release
     if: github.repository == 'remix-run/remix'
     runs-on: ubuntu-latest
+    outputs:
+      NEXT_VERSION: ${{ steps.version.outputs.NEXT_VERSION }}
```

this allows us to use this in the "comment" job below

---

Could you add a comment?

```suggestion
    outputs:
      # used in the comment job bellow
      NEXT_VERSION: ${{ steps.version.outputs.NEXT_VERSION }}
```